server/start/tryCatchForIndexPermissionError.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * Wazuh app - React HOCs to manage the message when elastic show a Response error
```

Comment:
It is not a `HOC`, it is a `HOF` (High order function)

Reply:
Thta's true, thanks you!